### ISSUE TITLE
low resolution on mac with GR 0.38 after update

### ISSUE THREAD

**Issue by floswald** [jheinen/GR.jl#201]
i just updated my package suite with `] up` and found that the plot window of GR now has really low resolution. how can i get back my super-crisp GR plots?

<img width="1030" alt="Screenshot 2019-03-20 12 47 38" src="https://user-images.githubusercontent.com/1021558/54682205-8edd3d80-4b0e-11e9-84bb-e28a5c247356.png">
<img width="608" alt="Screenshot 2019-03-20 12 49 42" src="https://user-images.githubusercontent.com/1021558/54682243-a7e5ee80-4b0e-11e9-927f-33497e19dd22.png">

**Comment by jheinen** [comment]
Which output is "low-res"? Which output device are you using or, what is you environment (Atom, IJulia, iTerm, Juno, ...) >

**Comment by floswald** [comment]
Mac OS terminal and plots come out on a GSKterm window.

**Comment by jheinen** [comment]
I can't really see where the problem is. The plots look fine for me!?

**Comment by floswald** [comment]
the same plot used to be razor sharp. as sharp as the "GSKTerm" labelling the window. i guess it's not a big deal, just was curious whether you made some changes related to that. I can live with this, so feel free to close.

**Comment by jheinen** [comment]
@FlorianRhiem : May this be related to the macOS version we are using for the CI build?

**Comment by FlorianRhiem** [comment]
The plot seems to be about half the expected resolution. I think I can reproduce the problem using a Retina display while the quality seems fine on my non-Retina external display, so Retina support would be my first guess. I'll look into it.

**Comment by floswald** [comment]
i confirm this is on retina display. I can only tell you that a little while ago that worked perfectly well. unfortunately i can't point you to any specific GR version, i havent kept track...

**Comment by FlorianRhiem** [comment]
I don't know why it worked before, it might have been due to some compatibility mechanism as we were using a very old macOS build system. I've found a fix for it, using the window's scale factor (e.g. 2 on retina displays and 1 on other displays) to scale the resolution that we draw in with GKSTerm. We will need to test whether this workaround has any major drawbacks and if it doesn't, it'll be part of the next release. Here's a comparison of text rendering before/after:
![image](https://user-images.githubusercontent.com/4411847/54982099-3db7c880-4faa-11e9-9265-83d33e2dad86.png)
![image](https://user-images.githubusercontent.com/4411847/54982112-45776d00-4faa-11e9-8167-ac3b3da93e90.png)

Thank you for pointing out this issue!
